src/js/components/reviewData/CertificationModal/RevalidateButtons.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+  * RevalidateButtons.jsx
+  * Created by Kevin Li 9/7/16
```

Comment:
Was this really created by Kevin last year sometime? Or is this name/date wrong?